Source/UI/CommonUI/UIStyle/CKOUIStyle+UIColor.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+extension CKOUIStyle {
+
+    enum Color {
+        enum Background {
+            static let primary = UIColor(hex: "#FF FF FF")
+        }
+
+        enum Text {
+            static let primary = UIColor(hex: "#00 00 00")
+            static let secondary = UIColor(hex: "#72 72 72")
+            static let actionPrimary = UIColor(hex: "#72 72 72")
+            static let actionSecondary = UIColor(hex: "#0B 5F F0")
+            static let disabled = UIColor(hex: "#72 72 72")
+            static let error = UIColor(hex: "#AD 28 3E")
+        }
+
+        enum Border {
+            static let primary = UIColor(hex: "#8A 8A 8A")
+            static let secondary = UIColor(hex: "#D9 D9 D9")
```

Comment:
Whilst I get why you'd separate RGB codes, I've never actually seen anyone breaking them up this way. Which would lead me to think any regular user of RGB codes (designers / engineers) would not necessarily derive any benefit from this either?

Reply:
Let me comment on myself 
IMHO, I would suggest to have `hex` colors `#E0FDAC` without spaces `#E0 FD AC` as it's common used.there's no right or wrong for both solutions. feel free to call me to discuss more
What is the severity of this change?  Nothing
What is the impact on merchants? Nothing
Is it normal format? Yes 
Is it a common used format? No

Why this format? 
Pros:
- Easy to read. 
- No need for RGB, Hex color is R G B in a Hexadecimal format. 
- No worries, I have seen it many times.

In the end, I will remove spaces.